components/builder-web/app/footer/footer.component.html

```diff
@@ -1,122 +1,65 @@
-<footer class="hab-footer">
-  <div class="footer--sitemap">
-    <div class="footer--logos">
-      <a href="{{config['www_url']}}" title="Habitat Home">
-        <img class="footer--logo habitat" src="../assets/images/habitat-logo-by-chef-white.svg" alt="Habitat by Chef" />
+<div class="footer-component">
+  <footer>
+    <div>
+      <a href="{{config['www_url']}}" title="Habitat.sh">
+        <img src="../assets/images/habitat-logo-by-chef-white.svg" alt="Habitat by Chef">
       </a>
     </div>
-    <div class="footer--links">
```

Comment:
I'm so stoked all the weird class names are gone.

Reply:
There's a system to that -- it's called [BEM](http://getbem.com/) and it's fairly common -- but I'm not a fan, either; I prefer just to use the cascade and to scope by nesting selectors. Some dislike this approach also ("never nest more than three levels deep!"), but I find it maps much more closely to the way I think about what I'm doing, and it hasn't let me down yet. 😄